packages/cli/src/WorkflowRunner.ts

```diff
@@ -191,6 +209,89 @@ export class WorkflowRunner {
 		return executionId;
 	}
 
+	async runBull(data: IWorkflowExecutionDataProcess, loadStaticData?: boolean, realtime?: boolean): Promise<string> {
+
+		// TODO: If "loadStaticData" is set to true it has to load data new on worker
+
+		// Register the active execution
+		const executionId = this.activeExecutions.add(data, undefined);
```

Comment:
In a concurrency environment, we will have multiple executions with the same ID as the current implementation here considers a static class and an internal counter. Once we merge the unification of execution ID, this probably won't be a problem anymore.

Adding this so other reviewers can know that it is being solved in another branch.

Reply:
Actually this would only happen if we start multiple "starters" which would not be the case at the moment. So my comment above can be ignored unless we split the webhook interceptors.

Reply:
As we merge with the unification branch before we split out the webhook-interceptors that should not be an issue.